requirements.txt

```diff
@@ -3,7 +3,7 @@
 broker[docker]==0.2.7
 cryptography==38.0.1
 deepdiff==5.8.1
-dynaconf[vault]==3.1.9
+dynaconf[vault]==3.1.11
 fauxfactory==3.1.0
 hvac<1.0.0  # hvac 1.0.0 breaks Dynaconf. #807
```

Comment:
@jyejare Would you mind removing this hvac workaround before you start testing the new update?

Reply:
@Gauravtalreja1 Probably that wont help as latest dynaconf version wont downgrade the hvac version. But still both the cases could be tested !

Reply:
@jyejare I agree with you, but now that dynaconf has been updated, I don't think we have to worry about hvac or any other dependencies anymore.
And fyi, @lpramuk opened a PR https://github.com/SatelliteQE/robottelo/pull/10103 already to remove this hvac workarounds

Reply:
oops I didn't realize we have dynaconf pinned in robottelo while in satellite6-upgrade is not